CMakeLists.txt

```diff
@@ -32,8 +33,12 @@ PKG_CHECK_MODULES(GLIB2 glib-2.0>=2.66 gio-2.0 REQUIRED)
 PKG_SEARCH_MODULE(LIBCRYPTO REQUIRED libcrypto openssl)
 PKG_CHECK_MODULES(LIBXML2 libxml-2.0 REQUIRED)
 FIND_PACKAGE(CURL 7.52.0 REQUIRED)
-FIND_PACKAGE(Gpgme REQUIRED)
 
+IF (USE_GPGME)
+    FIND_PACKAGE(Gpgme REQUIRED)
+ELSE (USE_GPGME)
+    PKG_CHECK_MODULES(RPM REQUIRED rpm>=4.18.0)
```

Comment:
I believe that we should add the conditional require to spec. It might be good to require virtual provide that will signal presence of rpm with right back-end - see https://github.com/rpm-software-management/rpm/issues/2556

Reply:
Thanks. I modified .spec file.